BadHabbit/BadHabbit/Kit/DesignKit/Models/Multipliers.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  Multipliers.swift
+//  BadHabbit
+//
+//  Created by Viktoria Misiulia on 05/09/2023.
+//
+
+import Foundation
+
+enum Multipliers {
+    static let phraseLabelLineHeight = 1.5
```

Comment:
why 1.5? we have the same value 1.26 on several screens

Reply:
value was changed